AdobeStockImageAdminUi/view/adminhtml/web/css/source/_module.less

```diff
@@ -79,4 +79,14 @@
             min-width: 70%;
         }
     }
+    .admin__field {
```

Comment:
Please do not use general selectors for styling concrete elements, rely only on the specific selector to the target element instead

Reply:
Hello @sivaschenko 
I've made sure that our adobe CSS won't break any style for core Magento elements as I've added label inside the tooltip but I've also added a new selector as per your requested changes.

Thanks